src/Lykke.Service.PayInternal.AzureRepositories/MerchantGroup/MerchantGroupEntity.cs

```diff
@@ -16,11 +17,11 @@ public class MerchantGroupEntity : AzureTableEntity, IMerchantGroup
 
         public string DisplayName { get; set; }
 
-        public string OwnerMerchantId { get; set; }
+        public string OwnerId { get; set; }
 
         public string Merchants { get; set; }
 
-        public AppointmentType Appointment { get; set; }
+        public GroupType Type { get; set; }
```

Comment:
GroupType is too general name. Rename it to MerchantGroupType or MerchantGroupPurpose.